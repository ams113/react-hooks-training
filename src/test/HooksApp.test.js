import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('Testing App: HooksApp', () => {

    test('should renders correctly', () => {

        const wrapper = shallow( <HooksApp />);
        expect( wrapper ).toMatchSnapshot();
    });

});