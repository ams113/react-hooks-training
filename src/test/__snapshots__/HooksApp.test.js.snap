// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App: HooksApp should renders correctly 1`] = `
<div>
  <h1>
    Hello world
  </h1>
</div>
`;
